src/modeler/AutoRest.Swagger/Validation/TrackedResourceGetOperationValidation.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.Core.Logging;
+using AutoRest.Core.Validation;
+using AutoRest.Swagger.Model.Utilities;
+using System.Collections.Generic;
+using AutoRest.Swagger.Model;
+using System.Text.RegularExpressions;
+using System.Linq;
+
+namespace AutoRest.Swagger.Validation
+{
+    public class TrackedResourceGetOperationValidation : TypedRule<Dictionary<string, Schema>>
+    {
+        private readonly Regex resNames = new Regex(@"(RESOURCE|TRACKEDRESOURCE)$", RegexOptions.IgnoreCase);
+
+        /// <summary>
+        /// The template message for this Rule. 
+        /// </summary>
+        /// <remarks>
+        /// This may contain placeholders '{0}' for parameterized messages.
+        /// </remarks>
+        public override string MessageTemplate => "Tracked resource {0} must have a get operation.";
```

Comment:
That probably is a good idea, but I see we're using Resources file from AutoRest.Core all over AutoRest swagger modeler tests, not sure if that's the right thing to do since these two are rather unrelated. Having said that, I'm on the fence about it.